SYCL/ESIMD/api/functional/ctors/ctor_broadcast.hpp

```diff
@@ -0,0 +1,196 @@
+//===-- ctor_broadcast.hpp - Functions for tests on simd broadcast constructor
+//      definition. -------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd broadcast constructor.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "common.hpp"
+
+namespace esimd = sycl::ext::intel::experimental::esimd;
+
+namespace esimd_test::api::functional {
```

Comment:
```suggestion
namespace esimd_test::api::functional::ctors {
```
What do you think?

Reply:
done in this commit: b12bfe332f0da75f23cdd49b29d10318fd4961db